setup.cfg

```diff
@@ -134,4 +133,4 @@ ignore =
     *-requirements.txt
 
 [tool:pytest]
-addopts = --black --isort --pydocstyle --ignore=docs --doctest-glob="*.rst" --doctest-modules --cov=invenio_communities --cov-report=term-missing
+addopts = -p no:warnings--black --isort --pydocstyle --ignore=docs --doctest-glob="*.rst" --doctest-modules --cov=invenio_communities --cov-report=term-missing
```

Comment:
Was that a typo? 

```suggestion
addopts = -p no:warnings --black --isort --pydocstyle --ignore=docs --doctest-glob="*.rst" --doctest-modules --cov=invenio_communities --cov-report=term-missing
```

Reply:
No, that was while testing other ways to make it pass as well. thanks for pointing it out!